src/mono/wasm/runtime-test.js

```diff
@@ -18,7 +31,7 @@ globalThis.testConsole = console;
 
 function proxyMethod (prefix, func, asJson) {
 	return function() {
-		var args = [...arguments];
+		var args = [...raw_args];
```

Comment:
is this correct?

Reply:
This is probably still wrong, I think it was `arguments` keyword